protected/views/patient/_form_address.php

```diff
@@ -60,8 +60,11 @@
 <div class="row field-row">
     <div class="large-3 column"><?php echo $form->labelEx($address,'country_id'); ?></div>
     <div class="large-4 column end">
-
-        <?php echo $form->dropDownList($address,'country_id', $countries, array('options'=>array(array_search('Australia', $countries)=>array('selected'=>true)))); ?>
+        <?php if (isset($address["country_id"])) {
```

Comment:
This code is from another commit/pull-request. You might have to be rebase this commit from the base branch locally, and force push back to this branch before you'll be able to rebase and merge

Reply:
Seems I messed up rebase things, I will create a new pull request.
I hate rebase :(